Makefile

```diff
@@ -5,9 +5,14 @@ vendor:
 	@GO111MODULE=on go mod tidy
 	@GO111MODULE=on go mod vendor
 
-build:
+build: build_om build_post_templates
+
+build_om:
 	@GO111MODULE=on go build -mod=vendor -o $(PWD)/om github.com/checkr/openmock/cmd/om
 
+build_post_templates:
+	@GO111MODULE=on go build -mod=vendor -o $(PWD)/om_post_templates github.com/checkr/openmock/cmd/post_templates
```

Comment:
since we are building more tooling around openmock, e.g. the debugging feature or gRPC mock generator in the future, does it make sense to start an `omctl` command? And post template can be a very good subcommand of `omctl`.

it's an open-source project, just want to avoid some breaking changes in the near future if we need to move `post_templates` to `omctl`.

Reply:
I think this is a great opportunity to introduce omctl as we start to add more functionality around tooling

Reply:
also, i'm curious what our approach is on testing command functionality?

Reply:
Agreed that we need more tests in openmock. We can run unit tests with golang interface mocks (if needed), and integration tests in a docker or docker-compose environment.